workalendar/tests/test_america.py

```diff
@@ -1,7 +1,10 @@
 from datetime import date
 
 from ..america import (Argentina, Barbados, Chile, Colombia, Mexico, Panama,
-                       Paraguay)
+                       Paraguay,)
```

Comment:
what's that change here? Brazil has its own test module...

Reply:
Actually i see this before, that brazil has it own module. I started to code on america, but i cut all to brazil module. This edit in truth dont change anything.

Reply:
@brunobord, i think that i solve the problems can you check again ? Thanks for instruct me, this is my first pr